File: YEAR2/SEM1/Metode Avansate de Programare/Seminar_06/src/domain/Entity.java
package domain;

public class Entity <ID> {
    private ID id;

    public Entity() {
        this.id = id;
    }

    public ID getId() {
        return id;
    }

    public void setId(ID id) {
        this.id = id;
    }

    @Override
    public String toString() {
        return "Entity [id=" + id + "]";
    }

}

File: YEAR2/SEM2/Medii de proiectare si programare/ConsoleApp1/domain/Entity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp1.domain
{
    [Serializable]
    public class Entity<ID>
    {

        public ID id { get; set; }

    }
}
